scripts/akv/key-import.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+akv-key-import() {
+    set -e
+
+    AKV_NAME=${AKV_NAME:-$1}
```

Comment:
AKV_VAULT_NAME

Reply:
Sure I'll do this

Reply:
Actually, doesn't the V in AKV already stand for vault?

Reply:
There a many AKV artificate AKV_CERTIFICATE_NAME, AKV_KEY_NAME

So it is clearer to state we declare the vault name in same way we do this for the certificate.

Reply:
AKV_NAME stands for Azure Key Vault Name
AKV_KEY_NAME stands for Azure Key Vault Key Name

Are you sure you want AKV_VAULT_NAME which stands for Azure Key Vault Vault Name? I think it's completely redundant, but it's not a big deal either way

Reply:
https://github.com/microsoft/azure-privacy-sandbox-kms/pull/233/commits/fa3f24570c49c3fb9538db1c25a5f5aa39658ae0